rllib/core/models/tests/test_cnn_encoders.py

```diff
@@ -84,10 +74,39 @@ def test_cnn_encoders(self):
             for fw in framework_iterator(frameworks=("tf2", "torch")):
                 # Add this framework version of the model to our checker.
                 outputs = model_checker.add(framework=fw)
-                self.assertEqual(outputs[ENCODER_OUT].shape, (1, output_dims[0]))
+                self.assertEqual(outputs[ENCODER_OUT].shape, (1, config.output_dims[0]))
 
             # Check all added models against each other.
-            model_checker.check()
+            model_checker.check(atol=0.0005)
+
+    def test_cnn_encoders_valid_padding(self):
```

Comment:
I separated this test, b/c our built-in utility function only produces auto-configs with "same" padding and I didn't want to mess up the original test.